bitnami/thanos/README.md

```diff
@@ -42,6 +42,24 @@ To uninstall/delete the `my-release` chart:
 helm uninstall my-release
 ```
 
+## Upgrading existing release to new major version
```

Comment:
In order to be consistent with other charts, we are used to placing this section at the end of the README:
```
 ## Upgrading

 ### To 1.0.0
...
```

Reply:
done